XamlAnimatedGif.Shared/Interfaces/ICacheHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿#region
+
+using System.IO;
+using System.Threading.Tasks;
+
+#endregion
+
+namespace XamlAnimatedGif.Interfaces
+{
+    internal interface ICacheHelper
```

Comment:
Did it based on the possibility of expanding each one of them, (like using the cache helper in another part), and re usability.  Of course, I could merge it all together :smile: